ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -27,7 +27,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```

Comment:
oc has wrong images `FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder` https://github.com/openshift/oc/blob/release-4.4/images/cli/Dockerfile.rhel#L1 you need to fix that first

Reply:
or just fix it here but the consistency with other configs will suffer a bit

Reply:
:facepalm:  yeah and that's across versions, opened https://github.com/openshift/oc/pull/494